device-types/Synology/RX1217RP.yaml

```diff
@@ -0,0 +1,18 @@
+---
+manufacturer: Synology
+model: RX1217RP
+slug: synology-rx1217rp
+part_number: RX1217RP
+u_height: 1.0
+is_full_depth: true
+weight: 13.4
+weight_unit: kg
+comments: Expansion Unit RX1217 with Redundant Power Supplies
+power-ports:
+  - name: PSU 1
+    type: iec-60320-c14
+  - name: PSU 2
+    type: iec-60320-c14
```

Comment:
The power for this chassis appears to be modular, is that correct?
If so, you should create `module-bays` on this device and then add the associated PSU definitions to the module-types folder

Reply:
ok, changes have been made